packages/react/src/patterns/blocks/ContentBlockSimple/__stories__/ContentBlockSimple.stories.js

```diff
@@ -0,0 +1,89 @@
+import './index.scss';
+import { select, text, withKnobs } from '@storybook/addon-knobs';
+import ContentBlockSimple from '../ContentBlockSimple';
+import React from 'react';
+import readme from '../README.md';
+import { settings } from 'carbon-components';
+import { storiesOf } from '@storybook/react';
+
+const { prefix } = settings;
+
+storiesOf('Patterns (Blocks)|ContentBlockSimple', module)
+  .addDecorator(withKnobs)
+  .addParameters({
+    readme: {
+      sidebar: readme,
+    },
+  })
+  .add('Default', () => {
+    const ctaProps = {
+      style: 'card',
+      type: 'external',
+      title: 'Lorem ipsum dolor sit amet',
+      href: 'https://www.ibm.com',
+      copy: 'Lorem ipsum dolor sit ametttt',
```

Comment:
I'd keep an eye on some of these to changes with @annawen1's pull request for high contrast.